apps/omg/lib/omg/signature.ex

```diff
@@ -42,6 +42,10 @@ defmodule OMG.Signature do
   This implements Eq.(208) of the Yellow Paper, adapted from https://stackoverflow.com/a/20000007
 
   """
```

Comment:
isn't this a great example for a doctest? https://elixir-lang.org/getting-started/mix-otp/docs-tests-and-with.html#doctests